### DISCUSSION TITLE
Where to host roadmaps of the tools/interactive layers etc?

### DISCUSSION THREAD

**Discussion by PRijnbeek** [EHDEN/ehden-roadmap#6] [Q&A]
Will there be separate projects per tool/solution, so for the service desk I create my own project here? and place major milestones in the EHDEN roadmap or do I use my own GitHub repository for my own tooling only bring the milestones backing the EHDEN-roadmap?

Advantage of having it all here is that you can oversee all the tools at one location? Disadvantage is you decouple the code from the roadmap.

**Comment by solventrix** [comment]
I think it makes sense to reserve this solely for the roadmaps where the different components are used as one of the features used in the tagging, as suggested by @MaximMoinat on the home page. This is a nice view on what we should also be able to do for our roadmap(s): https://github.com/github/roadmap/projects/1

**Comment by PRijnbeek** [comment]
Agreed then I will make a separate GitHub for the Service Portal with its own tool roadmap and the high level stays here.

see here the examples of the Superset Roadmap: https://github.com/apache-superset/superset-roadmap

See also their point on the EPICs/SIPS I like that too. Those are proposals. https://github.com/apache-superset/superset-roadmap#how-to-organize-the-content-in-the-roadmap

**Reply by PRijnbeek** [reply]
I made the [servicedesk repository](https://github.com/EHDEN/ServiceDesk), with its [roadmap](https://github.com/EHDEN/ServiceDesk/projects/1) and its own [discussion forum](https://github.com/EHDEN/ServiceDesk/discussions)
